cmd/cloud-controller-manager/main.go

```diff
@@ -46,6 +47,13 @@ import (
 )
 
 func main() {
+	if err := runCloudControllerManager(); err != nil {
```

Comment:
Please use a defer.

Reply:
the error they were trying to solve for is `os.Exit(1)` does not execute defer functions.